frontend/public/components/utils/_name-value-editor.scss

```diff
@@ -32,13 +46,17 @@
   padding-right: 8px;
 }
 
+.pairs-list__action-icon {
+  padding-right: 0;
+  width: auto;
+}
+
 .pairs-list__delete-icon {
-  line-height: 32px;
+  line-height: 35px;
 }
 
-.pairs-list__reorder-icon {
-  line-height: 32px;
-  padding-right: 10px;
+.pairs-list__action-icon--reorder {
```

Comment:
Nit:  we aim to alphabetize rules by selector, so this should move up to line 54.